.github/prepare-extension.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# This file is part of the phpBB Forum Software package.
+#
+# @copyright (c) phpBB Limited <https://www.phpbb.com>
+# @license GNU General Public License, version 2 (GPL-2.0)
+#
+# For full copyright and license information, please see
+# the docs/CREDITS.txt file.
+#
+set -e
+set -x
+
+EXTNAME=$1
+
+# Move the extension in place
+mkdir --parents phpBB/ext/$EXTNAME
+cp -R ../tmp/* phpBB/ext/$EXTNAME
+
+# Move the extensions .github/phpunit-*-travis.xml files in place
```

Comment:
It's not really checking for the word Travis

Reply:
That should probably be fixed, but for the Actions config I made for extensions we won't even be needing this file anymore, though it may be good to keep it around.